README.md

```diff
@@ -46,46 +46,86 @@ to it to verify how it behaves in the entire environment (end-to-end).
-It run a [collator](https://wiki.polkadot.network/docs/learn-collator) node:
+   It run a [collator](https://wiki.polkadot.network/docs/learn-collator) node:
     ```bash
     ./scripts/init.sh start-parachain
     ```
-    *Note: the command above will show logs and block until the parachain is stopped.
-    If you had a previous state, you can reset the node using `purge` after the command.*
+   *Note: the command above will show logs and block until the parachain is stopped.
+   If you had a previous state, you can reset the node using `purge` after the command.*
 
-    Similar to the relay chain, you can explore the parachain using the [polkadot.js (on localhost:11946)](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2Flocalhost%3A11946#/explorer) client.
-    You will see the block production frozen until you connect it to the relay chain.
+   Similar to the relay chain, you can explore the parachain using the [polkadot.js (on localhost:11946)](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2Flocalhost%3A11946#/explorer) client.
+   You will see the block production frozen until you connect it to the relay chain.
 
-    By default, the initialized parachain will have the id `2000`.
-    If you need more than one parachain or choose other chain specifications,
-    you can set `PARA_ID` or `PARA_CHAIN_SPEC`, example:
+   By default, the initialized parachain will have the id `2000`.
+   If you need more than one parachain or choose other chain specifications,
+   you can set `PARA_ID` or `PARA_CHAIN_SPEC`, example:
     ```bash
     PARA_ID=2001 ./scripts/init.sh start-parachain
     ```
-    The different `PARA_CHAIN_SPEC` values can be found at [`src/command.rs`](src/command.rs) under the `load_spec()` function.
+   The different `PARA_CHAIN_SPEC` values can be found at [`src/command.rs`](src/command.rs) under the `load_spec()` function.
 
 3. Onboard the parachain
-    This step will have the targeted parachain onboarded in the relay chain. The parachain will NOT produce blocks until this step is completed successfully.
+   This step will have the targeted parachain onboarded in the relay chain. The parachain will NOT produce blocks until this step is completed successfully.
     ```bash
     ./scripts/init.sh onboard-parachain
     ```
-    When you have run the command, you should see in the relay-chain dashboard that there is a parachain
-    that will be onboarded in one/two minutes.
-    Once onboarded, block production should start soon after in the parachain.
+   When you have run the command, you should see in the relay-chain dashboard that there is a parachain
+   that will be onboarded in one/two minutes.
+   Once onboarded, block production should start soon after in the parachain.
 
 That's all! The environment is set.
 You can play with it from the parachain client, make transfers, inspect events, etc.
 
+## Cargo tests
+
+To generate a corpus run:
+```shell
+cargo test
+```
+
+## Fuzz tests
```

Comment:
I think it should be in the next title level with three ### to be inside of the Test section